app/src/main/java/com/netmera/netmerafintech/ui/home/ManageCardActivity.kt

```diff
@@ -33,12 +36,14 @@ class ManageCardActivity: AppCompatActivity() {
         super.onCreate(savedInstanceState)
         binding = ActivityManageCardBinding.inflate(layoutInflater)
         setContentView(binding.root)
-
         val bundle: Bundle? = intent.extras
         bundle?.let {
             it.classLoader = ManageCardActivity::class.java.classLoader
         }
-        val card: Card? = bundle?.parcelable(ARG_CARD)
+
+        val card = intent?.data?.let { StaticData.getCards()[0] }
```

Comment:
Can you create a variable as `isComingFromDeeplink`. That would be more understandable.